### DISCUSSION TITLE
adblock format

### DISCUSSION THREAD

**Discussion by HanaO00** [qutebrowser/qutebrowser#6089] [Q&A]
Hello, can we hosts adblock list in hosts format with c.content.blocking.adblock.lists ?
Thanks

**Comment by The-Compiler** [comment]
No, that setting takes [ABP-like lists](https://adblockplus.org/en/filter-cheatsheet) only (at least I think so - maybe the underlying Brave adblock library thinks otherwise, so try it and see?).

However, you can add /etc/hosts-like lists to `content.blocking.hosts.lists` and set `content.blocking.method` to `both`.

**Comment by HanaO00** [comment]
In my config.py I did not put `content.blocking.method`, so that it remains on auto and I added several lists, but I didn't get any error so I don't know if it can read them anyway or not

**Reply by ninewise** [reply]
When you don't put `content.blocking.method` it's indeed on auto. If you don't have any ABP-like lists, qb should use your usual blocking lists. Try surfing to a domain on the list.
